types/react-native-navigation/react-native-navigation-tests.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Text, View } from 'react-native';
 import { Navigation, NavigationComponentProps, NavigatorStyle, NavigatorButtons } from 'react-native-navigation';
 
-class Screen1 extends React.Component<NavigationComponentProps<{ height: number }>> {
+class Screen1 extends React.Component<NavigationComponentProps & { height: number }> {
```

Comment:
Would you mind just creating a `Screen1Props` that extends `NavigationComponentProps` and adds a `height: number`? Interfaces tend to display better, verify more, and are checked more efficiently in general.